tests/conditional_compilation/conftest.py

```diff
@@ -99,4 +113,59 @@ def artifacts(request):
 @pytest.fixture(scope="session")
 def openvino_root_dir(request):
     """Fixture function for command-line option."""
-    return request.config.getoption("openvino_ref")
+    openvino_root_dir = request.config.getoption("openvino_root_dir")
+    validate_path_arg(openvino_root_dir, "openvino_root_dir", is_dir=True)
```

Comment:
see no reason to check "key isn't specified" in `validate_path_arg`, let's replace `getoption` with more strict version (like `request.config["openvino_root_dir"]` or smth like that)

Reply:
@vurusovs, note that `Config` object is not subscriptable